components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/handler/impl/OAuth2AccessTokenHandler.java

```diff
@@ -295,6 +325,26 @@ protected AuthenticationResult doAuthenticate(MessageContext messageContext) {
         return authenticationResult;
     }
 
+    /**
+     * If the application is MY_ACCOUNT and the token is impersonated allow only
+     * allowed operations during impersonation.
+     *
+     * @param resourceConfig Resource Config.
+     * @param allowedScopes  Allowed scopes in the token.
+     * @param clientId       Client id.
+     * @return Whether operation or not.
```

Comment:
Shall we update the comment to `@return Whether operation is allowed or not`

Reply:
Fixed with 397390dcc5069cbe6685873c77a02574393f3c33